src/sentry/eventstore/models.py

```diff
@@ -384,7 +428,8 @@ def get_grouping_variants(self, force_config=None, normalize_stacktraces=False):
 
     def get_primary_hash(self):
         # TODO: This *might* need to be protected from an IndexError?
-        return self.get_hashes()[0]
+        flat_hashes, _ = self.get_hashes()
```

Comment:
I'm not entirely sure. get_primary_hash is only used in unmerge afaict but I don't know yet how to handle the case where there are no flat hashes. I'd like to see it crash first so I understand it better. This will only happen for the mobile strategy.